components/PageCard.tsx

```diff
@@ -0,0 +1,158 @@
+import React, { useState } from 'react';
+import { Flex, Text } from '@100mslive/react-ui';
+
+import { DynamicIcon } from './Callout';
+import { ArrowRightIcon } from '@100mslive/react-icons';
+
+const PageCard = ({
+    showCta = true,
+    showTag = false,
+    showImage = false,
```

Comment:
Let's accept the image URL here and render that instead of the hardcoded link used. If this is not provided, we don't render the img element - same as above